ArjunaCore/arjuna/tests/classes/com/hp/mwtests/ts/arjuna/atomicaction/AtomicActionAsyncTest.java

```diff
@@ -49,6 +56,28 @@ public void testAbort() throws Exception {
     }
 
     @Test
+    public void testAbortAsyncAC() throws Exception {
```

Comment:
It is not obvious why the test validates that abort no longer produces an exception. Particularly so because the test commits the transaction (since you pass ActionStatus.COMMITTED to the executeTest method).